{
  "modified": "2024-06-25T13:32:33Z",
  "published": "2024-06-25T13:32:33Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-4786",
  "summary": "Malicious code in atio-obfuscator (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "atio-obfuscator",
        "purl": "pkg:pypi/atio-obfuscator"
      },
      "versions": [
        "0.1.4"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-03566",
        "import_time": "2024-06-28T02:48:18.281299524Z",
        "modified_time": "2024-06-25T13:32:33Z",
        "sha256": "6575976b23cca19538f01fa5759cf6d366f34cf8b7177351ae485b48ce621dae",
        "source": "reversing-labs",
        "versions": [
          "0.1.4"
        ]
      }
    ]
  }
}
